### ISSUE TITLE
ng build --prod error Expecting end of input, "RPAREN", "ADD", "SUB", "MUL", "DIV", got unexpected "FUNCTION"

### ISSUE THREAD

**Issue by xmlking** [akveo/nebular#2188]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [ ] feature request

### Issue description
`ng build --prod` produce this error. `ng build` works fine

 ```
[error] JisonParserError: Parse error on line 1: 
var(--calendar-width) + var(--calendar-weeknumber-width)var(--cale...
--------------------------------------------------------^
Expecting end of input, "RPAREN", "ADD", "SUB", "MUL", "DIV", got unexpected "FUNCTION"
    at /Users/schintha/Developer/Work/SPA/yeti/styles.55fcd96834b50b02d2e3.css:195:42
    at Parser.parseError (/Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss-calc/dist/parser.js:1164:15)
    at Parser.parse (/Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss-calc/dist/parser.js:1680:30)
    at /Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss-calc/dist/lib/transform.js:30:30
    at walk (/Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss-value-parser/lib/walk.js:19:7)
    at ValueParser.walk (/Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss-value-parser/lib/index.js:18:3)
    at transformValue (/Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss-calc/dist/lib/transform.js:24:50)
    at _default (/Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss-calc/dist/lib/transform.js:59:100)
    at /Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss-calc/dist/index.js:25:51
    at /Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss/lib/container.js:135:18
    at Rule.each (/Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss/lib/container.js:101:16)
    at Rule.walk (/Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss/lib/container.js:131:17)
    at /Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss/lib/container.js:148:24
    at Root.each (/Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss/lib/container.js:101:16)
    at Root.walk (/Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss/lib/container.js:131:17)
    at /Users/schintha/Developer/Work/SPA/yeti/node_modules/postcss-calc/dist/index.js:23:9
    at initializePlugin (/Users/schintha/Developer/Work/SPA/yeti/node_modules/cssnano/dist/index.js:31:51)
```

```
ng version     

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 9.0.0-rc.10
Node: 13.5.0
OS: darwin x64

Angular: 9.0.0-rc.11
... common, compiler, compiler-cli, core, forms
... language-service, localize, platform-browser
... platform-browser-dynamic, router, service-worker
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.803.23
@angular-devkit/build-angular     0.900.0-rc.10
@angular-devkit/build-optimizer   0.900.0-rc.10
@angular-devkit/build-webpack     0.803.23
@angular-devkit/core              8.3.23
@angular-devkit/schematics        8.3.23
@angular/animations               9.0.0-rc.10
@angular/cdk                      9.0.0-rc.8
@angular/cli                      9.0.0-rc.10
@angular/fire                     5.3.0
@ngtools/webpack                  9.0.0-rc.10
@schematics/angular               9.0.0-rc.10
@schematics/update                0.900.0-rc.10
rxjs                              6.5.4
typescript                        3.7.5
webpack                           4.41.2
```

sample project to reproduce 
https://github.com/xmlking/yeti

**Comment by ItsWendell** [comment]
I'm having the exact same error in version `5.0.0-beta.8`, in development mode it works fine. I also have the issue for some reason that once my css variables aren't enabled, the styling priorities 'changes'. In a few cases components are overwritten by theme styling instead of my locally defined styling. This was / is not the case with css variables enabled.

**Comment by yggg** [comment]
Thanks for reporting @xmlking and for digging into it @ItsWendell. After extracting calc terms into scss variables (#2274) project from the issue description compiles without errors. The fix is part of the 5.0.0 release, so closing for now. Feel free to reopen if you still experiencing the issue.
